scripts/trivia.js

```diff
@@ -410,75 +411,96 @@ TriviaGame.prototype.startNormalGame = function (points, cats, name) {
             }
         }
     }
+    var players = sys.playersOfChannel(triviachan);
+    // Flash players who have it enabled
+    var player_id, player_ip;
+    for (var p in players) {
+        player_id = players[p];
+        player_ip = sys.ip(player_id);
+        if (trivData.toFlash[player_ip]) {
+            // Lazy way to flash Webclient people since saying their name should flash.
+            // Probably a better way to do this.
+            if (sys.os(player_id) === "webclient") {
+                Trivia.sendPM(player_id, sys.name(player_id) + " a game is starting!", triviachan);
```

Comment:
Fuzzysqurl a game is starting!
Fuzzysqurl, a game is starting!

Comma makes it better imo